wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -443,24 +425,22 @@ class KotlinGenerator private constructor(
         .addFunction(encodeFun(type))
         .addFunction(decodeFun(type))
 
-    for (field in type.fields()) {
-      if (field.type().isMap) {
-        val keyType = field.type().keyType()
-        val valueType = field.type().valueType()
-        val adapterType = ProtoAdapter::class.asTypeName()
-            .parameterizedBy(Map::class.asTypeName()
-                .parameterizedBy(typeName(keyType), typeName(valueType)))
-        adapterObject.addProperty(
-            PropertySpec.builder("${field.name()}Adapter", adapterType, PRIVATE)
-                .initializer(
-                    "%T.newMapAdapter(%L, %L)",
-                    ProtoAdapter::class,
-                    getAdapterName(keyType),
-                    getAdapterName(valueType)
-                )
-                .build())
-      }
-    }
+    type.fields()
```

Comment:
I think the old version was fine and the new one is not necessarily better: although not really an issue here, functional operators create copies of the collection, so they're more memory-heavy and slower. I'd suggest reverting this piece.